docs/proposals/keeping-transient-pod-name-spaces-alive.md

```diff
@@ -0,0 +1,43 @@
+# Keeping transient pod name spaces alive
+
+## Introduction
+The goal of this proposal is to give administrators options to keep namespaces and therefore APB pods after the execution. The use cases for this feature are demos and debugging APBs.
+
+## Problem Description
+The [bug](https://bugzilla.redhat.com/show_bug.cgi?id=1497766) was created when we moved to creating transient namespaces during execution of the APB pod. What happens  This bug creates issues for debugging APB's as well as issues with demos. 
+
+## Implementation Details.
+
+### Configuration Values
+
+* `keep_namespace` or `BrokerConfig.KeepNamespace` - Parameter to always keep name space no matter what.
```

Comment:
I actually think that this should be in the `openshift`/clusterConfig section of the configuration.